estuary_updater/handlers/freshmaker.py

```diff
@@ -24,12 +24,31 @@ def can_handle(msg):
         :return: a bool based on if the handler can handle this kind of message
         :rtype: bool
         """
-        return msg['topic'] == '/topic/VirtualTopic.eng.freshmaker.event.state.changed'
+        if msg['topic'] == '/topic/VirtualTopic.eng.freshmaker.event.state.changed' or \
+                msg['topic'] == '/topic/VirtualTopic.eng.freshmaker.build.state.changed':
```

Comment:
Optional: you could change this to an `elif` to make it more readable. It's up to you which style you prefer.

Another option would be to create a list of topics and check if the topic is in that list, for instance:
```python
supported_topics = [
    '/topic/VirtualTopic.eng.freshmaker.event.state.changed',
    '/topic/VirtualTopic.eng.freshmaker.build.state.changed'
]

return msg['topic'] in supported_topics
```